api/sessions/add.js

```diff
@@ -0,0 +1,20 @@
+module.exports = addSession
+
+var createSession = require('../../utils/session/create')
+
+function addSession (state, options) {
+  return new Promise(function (resolve, reject) {
+    createSession({
+      couchUrl: state.url,
+      username: options.username,
+      password: options.password,
+      includeProfile: options.include === 'account.profile'
```

Comment:
do we need to mention in [here](http://docs.accountrestapi.apiary.io) that `?include=account.profile` is supported?